lib/active_triples/data_model.rb

```diff
@@ -0,0 +1,20 @@
+module ActiveTriples
+  ##
+  # Super class which provides a simple property DSL for defining property ->
+  # predicate mappings.
+  class DataModel
```

Comment:
Maybe this should be renamed?  I'm thinking something more like `PredicateModel` or `PredicateMapping` or some such?